pages/news.js

```diff
@@ -1,76 +1,77 @@
-import React from 'react';
-import styled from 'styled-components';
+import React, { useState } from 'react';
+
 import { NextSeo } from 'next-seo';
 import { useRouter } from 'next/router';
-import Link from 'next/link';
+
 import Layout from '../components/Layout';
 import Primary from '../components/Primary';
 import Sidebar from '../components/Sidebar';
 import NewsFeed from '../components/NewsFeed';
 import content from '../content/newsfeed.md';
+import Pagelinks from '../components/Pagelinks';
```

Comment:
Check this out real quick. If you update the filename to be capitalized, this should work then.

Reply:
Done , should be fine now